apps/api/src/server/applications/application/representations/status/__tests__/patch-status.test.js

```diff
@@ -1,19 +1,32 @@
 import { jest } from '@jest/globals';
 import { request } from '../../../../../app-test.js';
+import { eventClient } from '#infrastructure/event-client.js';
 
 const { databaseConnector } = await import('../../../../../utils/database-connector.js');
 
 const existingRepresentations = [
 	{
 		id: 1,
-		representationId: 1,
+		caseId: 1,
 		reference: 'BC0110001-2',
 		status: 'VALID',
 		redacted: true,
 		received: '2023-03-14T14:28:25.704Z'
+	},
+	{
+		id: 2,
+		caseId: 1,
+		reference: 'BC0110001-55',
+		status: 'PUBLISHED',
+		redacted: true,
+		received: '2023-08-11T10:52:56.516Z'
 	}
 ];
 
+const expectedUnpublishPayload = {
+	representationId: 2,
+	status: 'PUBLISHED'
```

Comment:
Should this status  be VALID ?

Reply:
Yes you are absolutely right - I was sending the old status not the newly updated one. I've amended the `publishRepresentationStatusUpdate` function to take the new status value as a parameter now